components/src/preact/lineageFilter/fetchLineageAutocompleteList.ts

```diff
@@ -1,20 +1,116 @@
+import { fetchLineageDefinition } from '../../lapisApi/lapisApi';
 import { FetchAggregatedOperator } from '../../operator/FetchAggregatedOperator';
 import type { LapisFilter } from '../../types';
 
 export async function fetchLineageAutocompleteList({
-    lapis,
-    field,
+    lapisUrl,
+    lapisField,
+    lapisFilter,
     signal,
+}: {
+    lapisUrl: string;
+    lapisField: string;
+    lapisFilter?: LapisFilter;
+    signal?: AbortSignal;
+}) {
+    const [countsByLineage, lineageTree] = await Promise.all([
+        getCountsByLineage({
+            lapisUrl,
+            lapisField,
+            lapisFilter,
+            signal,
+        }),
+        getLineageTree({ lapisUrl, lapisField, signal }),
+    ]);
+
+    return Array.from(lineageTree.keys())
+        .map((lineage) => {
+            return [
+                {
+                    lineage,
+                    count: countsByLineage.get(lineage) ?? 0,
+                },
+                {
+                    lineage: `${lineage}*`,
+                    count: getCountsIncludingSublineages(lineage, lineageTree, countsByLineage),
+                },
+            ];
+        })
+        .sort((a, b) => a[0].lineage.localeCompare(b[0].lineage))
```

Comment:
We can sort first. That seems a little easier.
```suggestion
        .sort((a, b) => a.localeCompare(b))
        .map((lineage) => {
            return [
                {
                    lineage,
                    count: countsByLineage.get(lineage) ?? 0,
                },
                {
                    lineage: `${lineage}*`,
                    count: getCountsIncludingSublineages(lineage, lineageTree, countsByLineage),
                },
            ];
        })
```